PIL/Image.py

```diff
@@ -1209,6 +1278,7 @@ def getextrema(self):
         return self.im.getextrema()
 
     def getim(self):
+        # type: () -> Any
```

Comment:
A PyCapsule is an internal python type: https://docs.python.org/3.6/c-api/capsule.html